### PR TITLE
openhv 20230917 (new cask)

### PR DESCRIPTION
**Pull request by Mailaender**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making any changes to a cask, existing or new, verify:

- [X] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [X] `brew audit --cask --online <cask>` is error-free.
- [X] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [X] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [X] Checked the cask was not [already refused](https://github.com/search?q=repo%3AHomebrew%2Fhomebrew-cask+is%3Aclosed+is%3Aunmerged+&type=pullrequests) (add your cask's name to the end of the search field).
- [X] `brew audit --cask --new <cask>` worked successfully.
- [X] `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --cask <cask>` worked successfully.
- [X] `brew uninstall --cask <cask>` worked successfully.

---

### CODE REVIEW

File: Casks/o/openhv.rb [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  sha256 "c7f0c0260f690d4039cf7146faad01bf1731b8b77a28eeb0cb9573200c5bc95f"
+
+  url "https://github.com/OpenHV/OpenHV/releases/download/#{version}/OpenHV-#{version}.dmg",
+      verified: "github.com/OpenHV/OpenHV/"
+  name "openhv"
+  desc "Pixelart Science-Fiction Real-Time-Strategy game"
+  homepage "https://www.openhv.net/"
+
+  livecheck do
+    regex(/(\d{8})/i)
```
Comment (bevanjkay) [inline Casks/o/openhv.rb:12]:
```suggestion
    url :url
    regex(/v?(\d+(?:\.\d+)*)/i)

File: Casks/o/openhv.rb [hunk 2]
```diff
@@ -0,0 +1,27 @@
+cask "openhv" do
+  version "20230917"
+  sha256 "c7f0c0260f690d4039cf7146faad01bf1731b8b77a28eeb0cb9573200c5bc95f"
+
+  url "https://github.com/OpenHV/OpenHV/releases/download/#{version}/OpenHV-#{version}.dmg",
+      verified: "github.com/OpenHV/OpenHV/"
+  name "openhv"
```
Comment (bevanjkay) [inline Casks/o/openhv.rb:7]:
```suggestion
  name "OpenHV"

File: Casks/o/openhv.rb [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+  url "https://github.com/OpenHV/OpenHV/releases/download/#{version}/OpenHV-#{version}.dmg",
+      verified: "github.com/OpenHV/OpenHV/"
+  name "OpenHV"
+  desc "Pixelart Science-Fiction Real-Time-Strategy game"
+  homepage "https://www.openhv.net/"
+
+  livecheck do
+    url :url
+    regex(/(\d{8})/i)
```
Comment (bevanjkay) [inline Casks/o/openhv.rb:13]:
```suggestion
    regex(/v?(\d+(?:\.\d+)*)/i)
```

The suggested regex follows our standard implementation.

File: Casks/o/openhv.rb [hunk 4]
```diff
@@ -0,0 +1,28 @@
+cask "openhv" do
+  version "20230917"
+  sha256 "c7f0c0260f690d4039cf7146faad01bf1731b8b77a28eeb0cb9573200c5bc95f"
+
+  url "https://github.com/OpenHV/OpenHV/releases/download/#{version}/OpenHV-#{version}.dmg",
+      verified: "github.com/OpenHV/OpenHV/"
+  name "OpenHV"
+  desc "Pixelart Science-Fiction Real-Time-Strategy game"
```
Comment (khipp) [inline Casks/o/openhv.rb:8]:
```suggestion
  desc "Pixel art science-fiction real-time strategy game"
```

File: Casks/o/openhv.rb [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+  app "OpenHV.app"
+
+  zap trash: [
+    "~/Library/Application Support/OpenRA/hv-news.json",
+    "~/Library/Application Support/OpenRA/Logs/irc.log",
+    "~/Library/Application Support/OpenRA/maps/hv",
+    "~/Library/Application Support/OpenRA/ModMetadata/hv-*",
+    "~/Library/Application Support/OpenRA/Replays/hv",
+    "~/Library/Application Support/OpenRA/Saves/hv",
+    "~/Library/Application Support/OpenRA/Screenshots/hv",
```
Comment (khipp) [inline Casks/o/openhv.rb:26]:
```suggestion
    "~/Library/Application Support/OpenRA/hv-news.json",
    "~/Library/Application Support/OpenRA/Logs",
    "~/Library/Application Support/OpenRA/maps",
    "~/Library/Application Support/OpenRA/ModMetadata",
    "~/Library/Saved Application State/net.openra.mod.hv.savedState",
```

It seems like replays, saves, and screenshots are user-created data, so we should keep them.

**Review by bevanjkay — APPROVED** [review]

**Pull request merged**